internal/api/controllers/private/highlevelConnectionStatus.go

```diff
@@ -33,6 +33,14 @@ func (this *controllers) ApiInternalHighlevelConnectionStatus(ctx echo.Context)
 		return ctx.NoContent(http.StatusBadRequest)
 	}
 
+	if len(input.Hosts) > 50 {
+		utils.GetLogFromEcho(ctx).Infow("More than 50 hosts requested")
+
+		return ctx.JSON(http.StatusBadRequest, map[string]string{
+			"message": "maximum input length exceeded",
```

Comment:
What do you think about changing this to something like "Maximum number of hosts exceeded"?  This might make it a bit more clear about why the request failed.

This is kinda nit picky.  Sorry!